h/static/scripts/test/admin-users-test.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+var AdminUsersController = require('../admin-users');
+
+describe('CreateGroupFormController', function () {
+  var element;
+  var template;
+
+  before(function () {
+    template = '<form id="js-users-delete-form">' +
+               '<input type="submit">';
+  });
+
+  beforeEach(function () {
+    element = document.createElement('div');
+    element.innerHTML = template;
+  });
+
+  it('it should confirm the delete user form submission', function () {
+    // click on submit button
+    // assert that confirm is being called
```

Comment:
There are two basic ways to stub functions like `window.confirm`:

Instead of relying on the `window` global in the implementation, inject it. Then in tests you can inject a stub, eg:

```
function AdminUsersController(element, window_) {
...
window_.confirm(...))
}
```

Then in the tests:

```js
var fakeWindow = {confirm: sinon.stub().returns(true)};
var controller = new AdminUsersController(element, fakeWindow)
...
assert.called(fakeWindow.confirm)
```

The other option is to rely on Sinon's ability to stub globals using `sinon.stub(object, function name)` in `beforeEach()` and then remove the stub in `afterEach()` using `object.functionName.restore();`

I generally prefer the first approach as it is a little more straightforward.